packages/lesswrong/server/users/cron.ts

```diff
@@ -0,0 +1,35 @@
+import { addCronJob } from '../cronUtil';
+import Users from "../../lib/collections/users/collection";
+import { ModeratorActions } from "../../lib/collections/moderatorActions";
+import { allRateLimits } from "../../lib/collections/moderatorActions/schema";
+import { getSignatureWithNote } from "../../lib/collections/users/helpers";
+import * as _ from 'underscore';
+import moment from 'moment';
+
+
+addCronJob({
+  name: 'expiredRateLimitsReturnToReviewQueue',
+  interval: 'every 24 hours',
+  async job() {
+  
+  
+    const startOfDay = moment(new Date()).subtract(1, 'days').toDate()
+    const endOfDay = new Date() 
```

Comment:
nit: to reliably get an exact 24-hour window, probably declare `endOfDay` first, then use `endOfDay` to compute `startOfDay` instead of calling `new Date()` again